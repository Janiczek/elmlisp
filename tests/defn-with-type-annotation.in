(defn foo : (-> Int String)
  (bar)
  "haha")
