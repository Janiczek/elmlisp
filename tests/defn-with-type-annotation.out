foo : Int -> String
foo bar =
    "haha"
